skyline-vscode/react-ui/src/App.js

```diff
@@ -137,7 +139,7 @@ function App() {
 
     useEffect(function () {
         window.addEventListener('message', event => {
-            console.log("Message:", JSON.stringify(event.data));
+            //console.log("Message:", JSON.stringify(event.data));
```

Comment:
Delete instead of commenting out

Reply:
Done, instead added other console messages for each type of event (received data, text changed, and error)